build_package.py

```diff
@@ -29,3 +32,20 @@
     shutil.rmtree(_egg_dir)
 
 subprocess.run("python setup.py sdist bdist_wheel", shell=True)
+
+base_wheel_location = glob.glob("dist/*.whl")[0]
+without_platform = base_wheel_location[:-7]
+
+pack_data = [
+    ("driver-linux", "manylinux1_x86_64.whl"),
+    ("driver-macos", "macosx_10_13_x86_64.whl"),
+    ("driver-win.exe", "win_amd64.whl"),
+]
+
+for driver, wheel in pack_data:
+    wheel_location = without_platform + wheel
+    shutil.copy(base_wheel_location, wheel_location)
+    with zipfile.ZipFile(wheel_location, "a") as zipf:
+        driver_location = f"playwright/drivers/{driver}.gz"
```

Comment:
We no longer need intermediate gzip then, the only reason behind it was to not extract stray platform files, now we don't have them. So this and main.py can change.